test/e2e/specs/test.spec.js

```diff
@@ -1,52 +1,168 @@
+            cy.contains('button', 'Withdraw').click();
+            cy.contains('.input-label', 'Amount')
+              .next('.input-wrapper')
+              .within(() => {
+                cy.get('input[type="number"]').click();
+                cy.get('input[type="number"]').type(5);
+              });
+          });
+
+        cy.contains('button', 'Adjust Vault').click();
+        cy.confirmTransaction().then(taskCompleted => {
           expect(taskCompleted).to.be.true;
+          cy.contains('p', "Your vault's balances have been updated.", {
+            timeout: MINUTE_MS,
+          }).should('exist');
+          cy.contains('Back to my vaults').click();
         });
-      });
+      }
     });
 
-    it('should adjust the collateral by performing a withdrawl and approve the transaction successfully', () => {
+    it('should create a new vault and approve the transaction successfully', () => {
       cy.visit('/');
-      cy.contains('div', /ATOM.*#5/).click();
+      if (!networkPhrases.isLocal)
+        cy.get('button').contains('Keep using Old Version').click();
+
+      if (networkPhrases.isLocal)
+        cy.contains('button', 'Add new vault').click();
+      cy.contains('button', /ATOM/).click();
+
+      cy.contains('.input-label', 'ATOM to lock up *')
+        .next()
+        .within(() => {
+          cy.get('input[type="number"]').click();
+          cy.get('input[type="number"]').clear();
+          cy.get('input[type="number"]').type(2);
+        });
+
+      cy.contains('button', 'Create Vault').click();
+
+      cy.confirmTransaction().then(taskCompleted => {
+        expect(taskCompleted).to.be.true;
+        cy.contains(
+          'p',
+          'You can manage your vaults from the "My Vaults" view.',
+          { timeout: MINUTE_MS },
+        ).should('exist');
+        cy.contains('Manage my Vaults').click();
+      });
+    });
+
+    it('should open the new vault', () => {
+      cy.get('span')
+        .contains(/My Vaults.*\(\d+\)/)
+        .children()
+        .first()
+        .spread(element => {
+          // Get the total number of vaults present
+          const vaultCountRegex = element.innerHTML.match(/\((\d+)\)/);
+          const vaultCount = Number(vaultCountRegex[1]);
 
+          cy.findAllByText(/#\d+/)
+            .should('have.length', vaultCount)
+            .spread((...vaults) => {
+              expect(
+                vaults.filter(vaultNo => !/^#\d+$/.test(vaultNo.innerHTML)),
+              ).to.be.empty;
+              // Get the vault with the largest number and click on it
+              const maxValue = vaults.reduce(
+                (maxValue, currentValueRaw) => {
+                  const currentValue = Number(
+                    currentValueRaw.innerHTML.slice(1),
+                  );
+                  return currentValue > maxValue ? currentValue : maxValue;
+                },
+                Number(vaults[0].innerHTML.slice(1)),
+              );
```

Comment:
If I'm following your code right, this would be clearer:
```suggestion
              const vaultIds = vaults.map(v => Number(v.innerHTML.slice(1)));
              const latestId = Math.max(...vaultIds);
```